src/app/testing/jest/frontend/components/cards/CollegeCard.test.tsx

```diff
@@ -44,8 +44,8 @@ describe("Test CollegeCard", () => {
     );
   });
 
-  test("CollegeCard button group should have one button", () => {
-    expect(screen.getAllByRole("button")).toHaveLength(1);
+  test("CollegeCard button group should have two buttons", () => {
+    expect(screen.getAllByRole("button")).toHaveLength(2);
```

Comment:
(need to update this too if we remove more info)

Reply:
Fixed :D